project/frontend/src/components/ResultsMap.js

```diff
@@ -8,34 +8,47 @@ const mapStyle = {
 };
 
 function MapContainer(props) {
-  const [activeMarker, changeMarker] = useState({});
-  const [showInfoWindow, changeInfoVisibility] = useState(false);
-
-  const coords = { lat: 40.837, lng: -73.865 };
+  const [activeMarker, setActiveMarker] = useState({});
+  const [showInfoWindow, setShowInfoWindow] = useState(false);
+  // TODO: add a conditional to check if the list is empty
+  const restaurant = props.restaurant;
+  const matchCoords = restaurant[0].key.latLngCoords;
 
   const onMarkerClick = (props, marker) => {
-    changeMarker(marker);
-    changeInfoVisibility(true);
+    setActiveMarker(marker);
+    setShowInfoWindow(true);
   };
 
   const onClose = (props) => {
     if (showInfoWindow) {
-      changeInfoVisibility(false);
+      setShowInfoWindow(false);
+    }
+  };
+
+  const createMarkers = () => {
+    let markers = [];
+    const numOfMarkers = 4;
+    for (let i = 0; i < numOfMarkers; i++) {
```

Comment:
I think we should change numOfMarkers to min (restaurant.length, 4) -- as restaurant[i] will be not a valid index if we happen to have < 4 restaurants